app/terminal/terminal.tsx

```diff
@@ -263,6 +264,12 @@ export default class TerminalComponent extends React.Component<TerminalProps, St
   }
 
   private scrollToEnd() {
+    let lineNumber = router.getLineNumber();
+    if (lineNumber) {
+      this.scrollToRow(lineNumber);
+      return;
+    }
```

Comment:
nit: I feel like it might be a decent idea to add line numbers to the logs at some point. Line numbers are usually 1-indexed, so wdyt about 1-indexing these instead of 0-indexing so that the URL param matches the selected line in that case?